tests/integration/workflows/fixtures/oai_arxiv_record_with_plots.xml

```diff
@@ -24,7 +24,7 @@
             </author>
           </authors>
           <title>Spectra generated by a confined softcore Coulomb potential</title>
-          <categories>math-ph math.MP</categories>
+          <categories>math-ph math.MP astro-ph astro-ph.IM</categories>
```

Comment:
> That it does not apply to this PR

Just for clarity: mine was a comment, not something intended to block this PR. In fact, I agree on all of your points, when you are being intellectually honest fake or real records don't really matter. The problem is that it's very easy to lapse in judgement, because "[...] you must not fool yourself – and you are the easiest person to fool". If you want a concrete example inside of the code base, look at the tests for `contact_details` in `tests/unit/dojson`.